spec/unit/puppet/provider/dhparam/openssl_spec.rb

```diff
@@ -10,30 +12,22 @@
 
   context 'when using default size' do
     it 'creates dhpram with the proper options' do
-      expect(provider_class).to receive(:openssl).with([
-                                                         'dhparam',
-                                                         '-out', '/tmp/dhparam.pem',
-                                                         512
-                                                       ])
+      expect(provider_class).to receive(:openssl).with(['dhparam', '-out', '/tmp/dhparam.pem', 512]) # rubocop:disable RSpec/MessageSpies
       resource.provider.create
     end
   end
 
   context 'when setting size' do
     it 'creates dhpram with the proper options' do
       resource[:size] = 2048
-      allow(provider_class).to receive(:openssl).with([
-                                                        'dhparam',
-                                                        '-out', '/tmp/dhparam.pem',
-                                                        2048
-                                                      ])
+      allow(provider_class).to receive(:openssl).with(['dhparam', '-out', '/tmp/dhparam.pem', 2048])
       resource.provider.create
     end
   end
 
   it 'deletes file' do
-    expect(Pathname).to receive(:new).twice.and_return(pathname)
-    expect(pathname).to receive(:delete)
+    expect(Pathname).to receive(:new).twice.and_return(pathname) # rubocop:disable RSpec/MessageSpies
```

Comment:
Should we disable message spies in the whole module for now?

Reply:
I added a rubocop_todo.yml where RSpec/StubbedMock and RSpec/MessageSpies are disabled.